utilities/after_success.sh

```diff
@@ -10,6 +10,29 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
     mvn cobertura:cobertura coveralls:report
     mvn site-deploy -DskipTests=true --settings=target/travis/settings.xml
     mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
+
+    # Update "site/latest/index.html" to redirect to the newly generated website
+    git config --global user.name "Travis CI"
+    git config --global user.email ${CI_DEPLOY_USERNAME}@github.com
+    mkdir tmp_gh-pages # Avoid clashing filenames when cloning repo
+    cd tmp_gh-pages
+    git clone https://github.com/GoogleCloudPlatform/gcloud-java.git
+    cd gcloud-java
+    SITE_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev '(^\[|Download\w+:)')
+    SITE_VERSION=${SITE_VERSION%-*} # Strip "-SNAPSHOT" out of version as necessary
+    git checkout gh-pages
+    mkdir -p site/latest/
+    touch site/latest/index.html
+    echo "<html><head><meta http-equiv=\"refresh\" content=\"0; URL='http://GoogleCloudPlatform.github.io/gcloud-java/site/${SITE_VERSION}/index.html'\" /></head><body></body></html>" > site/latest/index.html
+    git add site/latest/index.html
+
+    # Update "Quickstart with Maven" block on landing page to reflect latest version
+    sed -i "s/SITE_VERSION/$SITE_VERSION/g" site/${SITE_VERSION}/index.html
+    git add site/${SITE_VERSION}/index.html
+
+    git commit -m "Updating to reflect latest website version"
+    git config --global push.default simple
+    git push --force --quiet "https://${CI_DEPLOY_USERNAME}:${CI_DEPLOY_PASSWORD}@github.com/GoogleCloudPlatform/gcloud-java.git" > /dev/null 2>&1
```

Comment:
Do you really want to `--force` push here? If you have multiple simultaneous builds that will cause problems.